src/features/user-dashboard/pages/UpdateUserView.tsx

```diff
@@ -1,39 +1,98 @@
+import { Card, Modal } from 'react-bootstrap';
 import { PageCrumb, PageHeader, SmallContainer } from 'common/styles/page';
 import { useRbac } from 'features/rbac';
 import { FC, useEffect, useState } from 'react';
 import { Link, useNavigate, useParams } from 'react-router-dom';
 import { FormData, UserDetailForm } from '../components/UserDetailForm';
 import { Trans } from 'react-i18next';
+import { ChangeLog } from 'common/components/ChangeLog/ChangeLog';
+import { useAuth } from 'features/auth/hooks';
+import { LoadingButton } from 'common/components/LoadingButton';
+import { useInfiniteLoading } from 'common/hooks/useInfiniteLoading';
+import { HistoricalRecord } from 'common/models/historicalRecord';
+import { QueryParamsBuilder } from 'common/api/queryParamsBuilder';
+import { ChangeListGroup } from 'common/components/ChangeLog/ChangeListGroup';
+import { useModal } from 'react-modal-hook';
 
 type RouteParams = {
   id: string;
 };
 
 export const UpdateUserView: FC = () => {
   const { id = '' } = useParams<RouteParams>();
+  const { user: loggedInUser } = useAuth();
   const navigate = useNavigate();
   const { userHasPermission } = useRbac();
   const [updateUser] = useUpdateUserMutation();
   const { data: user, isLoading: isLoadingUser, isFetching, error: getUserError } = useGetUserByIdQuery(id);
+
+  const pageSize = 5;
+  const queryParams = new QueryParamsBuilder().setPaginationParams(1, pageSize).build();
+  const url = `/users/${id}/history/?${queryParams}`;
+  const {
+    loadedData: userHistory,
+    error: userHistoryError,
+    isFetching: isFetchingHistory,
+    totalCount,
+    hasMore,
+    fetchMore,
+  } = useInfiniteLoading<HistoricalRecord<User>, PaginatedResult<HistoricalRecord<User>>>(url, useGetUserHistoryQuery, {
+    skip: loggedInUser?.role !== 'ADMIN',
+  });
+
   const roles = Object.values(Role);
 
   const availableRoles = roles.filter(role => userHasPermission({ permission: 'role:read', data: role }));
   const [formValidationErrors, setFormValidationErrors] = useState<ServerValidationErrors<FormData> | null>(null);
 
+  const [showModal, hideModal] = useModal(
+    ({ in: open, onExited }) => {
+      return (
+        <Modal show={open} onHide={hideModal} onExited={onExited}>
+          <Modal.Header closeButton>
+            <Modal.Title>Changes</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <ChangeListGroup changeList={userHistory} />
+          </Modal.Body>
+          {hasMore && (
+            <Modal.Footer className='d-flex justify-content-center'>
+              <LoadingButton
+                className='action-shadow'
+                loading={isFetchingHistory}
+                variant='primary'
+                onClick={() => fetchMore()}
+              >
+                Load More
+              </LoadingButton>
+            </Modal.Footer>
+          )}
+        </Modal>
+      );
+    },
+    [userHistory, isFetchingHistory],
+  );
```

Comment:
Lets give this modal a max height and make it so that it can scroll inside the modal itself rather than the whole page scrolling.

![image](https://user-images.githubusercontent.com/2287977/202551923-55977e21-0889-4e11-bc26-a6e5d7b4b625.png)